src/cmd/sv-rollout/svrestarter.go

```diff
@@ -102,7 +107,7 @@ func (s *SvRestarter) notifyResult(result error) {
 	case ErrRestartPreempted:
 		s.log("was not required to restart in time", true)
 	default:
-		panic(result)
+		s.log(fmt.Sprintf("Unexpected error handled, likely a bug: %s : %s", reflect.TypeOf(result).String(), result), true)
```

Comment:
Not sure if failing deploys for this is okay. Without preempt it should fail anyways and with it we dont care (according to original PR).